app/models/service_template_ansible_playbook.rb

```diff
@@ -93,6 +93,7 @@ def self.build_parameter_list(name, description, info)
       :playbook                 => playbook.name,
       :inventory                => tower.provider.default_inventory,
       :become_enabled           => info[:become_enabled].present?,
+      :verbosity                => info[:verbosity] || 0,
```

Comment:
@gmcculloug - Yeah - I did more testing - I thought an empty string would be ok from the API side - but it blows up.  I'll get that change in.